translations/messages.en.yaml

```diff
@@ -40,3 +40,7 @@ ngsite.job_application.responsibilities: 'Responsibilities'
 ngsite.job_application.deadline: 'Application deadline'
 
 ngsite.skip_to_main_content: 'Skip to main content'
+
+ngsite.share_on_twitter: 'Share on Twitter'
+ngsite.share_on_linkedin: 'Share on LinkedIn'
+ngsite.share_on_facebook: 'Share on Facebook'
```

Comment:
Missing new line at EOF.

Reply:
fixed in https://github.com/netgen/media-site/pull/214/commits/31049da141cfa1c807517ef467ae23ea02e929fe